common.py

```diff
@@ -163,11 +189,20 @@ class IPRouteRouteEntry:
     dev: str
 
 
-def ipr_to_entries(input: str) -> list[IPRouteRouteEntry]:
-    j = json.loads(input)
+def ipr_to_entries(jstr: str, *, strict_parsing: bool = False) -> list[IPRouteRouteEntry]:
     ret: list[IPRouteRouteEntry] = []
-    for e in j:
-        ret.append(IPRouteRouteEntry(e["dst"], e["dev"]))
+    for e in _json_load(jstr, strict_parsing=strict_parsing):
+        try:
+            entry = IPRouteRouteEntry(
+                e["dst"],
+                e["dev"],
+            )
+        except (KeyError, ValueError, TypeError):
```

Comment:
Before this commit, the function would error out on ValueError, but your code now skips those lines in case of `strict_parsing = False`.  I don't see why you would want to skip those values. Can you elaborate?